mw-webapp/src/pages/usersPage/UsersPage.tsx

```diff
@@ -0,0 +1,7 @@
+/**
+ * Users page will contain all user tables and the filter/search panel
```

Comment:
You violate encapsulation here and also brings
Duplication

Reply:
Done